toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/tests/apps/test_fio.py

```diff
@@ -0,0 +1,17 @@
+from pathlib import Path
+
+from cijoe.fio.wrapper import fio_fancy
+
+from ..conftest import xnvme_parametrize
+
+
+@xnvme_parametrize(labels=["dev"], opts=["be", "admin", "sync", "async"])
+def test_fio_engine(cijoe, device, be_opts, cli_args):
```

Comment:
True, the unused ``cli_args`` is not pretty, however, it is provided since ``xnvme_parametrize()`` produces the list of ``(device, be_opts, cli_args)``, if the argument is removed from the test-func, then it fails.

Reply:
Okay in that case, then `cli_args` are missing from the tests in `test_cython.py`

Reply:
Oh, i might be wrong then! Let me just check

Reply:
Hmm, I am seeing ``function uses no argument 'cli_args'``, when unused is left out.

Reply:
Ohh, it is skipped, that is why it is not failing with missing argument...

Reply:
Fixed!

Reply:
And now, style-issue fixed.

Reply:
And, now the tests ought to pass.